wiki/History_of_osu!/2010/en.md

```diff
@@ -1 +1,68 @@
+
+![A touchscreen displaying osu!'s main menu](img/osu!arcade.jpg "The osu!arcade machine at 2010's Wai-con")
+
+## February
+
+A new osu! update made small improvements in the overall user interface and added the ability to move all storyboard events by an arbitrary time value, allowing for more fine-tuned storyboard animations. In addition to this, usernames became clickable in chat logs.[^stable-b1460]
+
+## March
+
+Through February and March, [peppy](https://osu.ppy.sh/users/2) organised an art competition about drawing banners for in-game challenges. After a [community voting](https://osu.ppy.sh/community/forums/topics/27112?n=1) concluded, the osu! team picked the final selection. The winner's art being officially used as well as one year of [osu!supporter](/wiki/osu!supporter) or physical goods being awarded were in prospect for the competition's finalist.[^art-challenge]
+
+On March 8, an osu! update brought mainly performance and audio improvements to the game.[^stable-b1485]
+
+## April
+
+Due to infrastructure changes, the [IRC](https://en.wikipedia.org/wiki/Internet_Relay_Chat) was moved to the address `irc.ppy.sh`.[^irc]
+
+This month, peppy introduced a new star rating difficulty algorithm in an osu! update.[^stable-b1596]
+
+## August
+
+Since August 2, users became able to change their username once. To change their name, they had to have received an osu!supporter tag at least once beforehand, either through purchase or as a gift.[^name-change]
+
+[Bancho](wiki/BanchoBot) received the [`!report`](/wiki/de/BanchoBot#report) command, allowing people to notify chat moderators of users that showed unsolicited behaviour (e.g. spamming in chat).[^bancho-report]
+
+[nardii](https://osu.ppy.sh/users/1017) gave peppy access to their [Twitter](https://twitter.com) account [@osugame](https://twitter.com/osugame) which was used for general osu! news from then on.[^twitter-osugame]
+
+## September
+
+On September 2, osu! received several mapping updates. Among those were a display for showing how efficiently storyboard and background elements are drawn on the screen, known as [SB load](/wiki/Client/Beatmap_editor/SB_load). Additionally, users became able to set the speed of [approach circles](/wiki/Gameplay/Hit_object/Approach_circle) regardless of [overall difficulty](/wiki/Beatmap/Overall_difficulty). Slider speeds between 2x and 1/2x became able to be adjusted to custom values.[^stable-b1650]
+
+60 [difficulties](/wiki/Beatmap/Difficulty) were affected by a Taiko bug causing drum-rolls to be incorrectly displayed and thus, all scores on them were deleted.[^taiko-reset]
+
+## October
```

Comment:
I'm not sure how detailed the `Flashlight` mod fiasko through October and November should be mentioned here. To reiterate what happened, take a look at these threads:

**October:**

- [flashlight mod deactivated](https://osu.ppy.sh/community/forums/topics/38692?n=1)
- [flashlight is back](https://osu.ppy.sh/community/forums/topics/38760?n=1)

**November:**

- [flashlight disabled again](https://osu.ppy.sh/community/forums/topics/41039?n=1)
- [flashlight is back (again)](https://osu.ppy.sh/community/forums/topics/41519?n=1)

Reply:
I like the idea, but sadly I don't recall any similar osu!-related issues having being posted on the wiki

Reply:
The `Flashlight` mod page already mentions that the mod had to be deactivated due to those bugs.

Also, there is the ongoing discussion of what to do with these history sections spread around the wiki.

I guess we can leave it be for now and just have this comment as reference for later.

Reply:
I think it would be nice to document it.